src/plugins/data/public/index_patterns/index_patterns/index_pattern.ts

```diff
@@ -153,11 +153,17 @@ export class IndexPattern implements IIndexPattern {
     if (!this.id) {
       return;
     }
-
-    if (forceFieldRefresh || this.isFieldRefreshRequired()) {
-      await this.refreshFields();
+    try {
+      if (forceFieldRefresh || this.isFieldRefreshRequired()) {
```

Comment:
Couldn't you instead, **not throw** the error from `refreshFields`?
I think it should create the same effect, no?

Reply:
thx! this is definitely the better solution! especially with the comment you've mentioned, a warning about not to do what happens 10 lines after the warning 😄